lib/utils/latest_version.rb

```diff
@@ -1,19 +1,13 @@
 # author: Christoph Hartmann
 
 require 'json'
-require 'net/http'
+require 'open-uri'
 
 class LatestInSpecVersion
   # fetches the latest version from rubygems server
   def latest
     uri = URI('https://rubygems.org/api/v1/gems/inspec.json')
-    res = Net::HTTP.start(
-      uri.host, uri.port, use_ssl: uri.scheme == 'https',
-      open_timeout: 0.5, read_timeout: 0.5
-    ) { |http|
-      http.get(uri.path)
-    }
-    inspec_info = JSON.parse(res.body)
+    inspec_info = JSON.parse(uri.read)
```

Comment:
Because InSpec is often executed in airgapped environments, we need to fail fast when there is no hope of a network connection - hence the specification of timeouts. I think we need to retain that.

Reply:
I wonder if we should just rip this functionality out entirely. I'd rather we have a command that checks if we're outdated and version just reporting the version; quickly.

Considering this feature is based around the version of the Gem, and we really want people to use chefdk / omnibus versions. If its a feature to be left in, we should be querying the latest omnibus version.

`inspec outdated` (probably pick a better name) - reports whether the user has a outdated version of inspec, what the latest version of inspec is, and the url to download it.

Reply:
I could see `inspec outdated`, as you suggest, but that would mean breaking change in `inspec version`'s behavior. What about adding `inspec version --no-check`?